content/docs/use-cases/ci-and-ml.md

```diff
@@ -9,11 +9,11 @@ versioning control. Another adoption, unfortunately less common though, could be
 unit testing the various stages of a data processing pipeline. Here is a non
 exhaustive list of a few other unique challenges:
 
-- automatically re-train models upon changes
-- systematically compare (and report) performance of different versions of a
+- Automatically re-train models upon changes
+- Systematically compare (and report) performance of different versions of a
   model
-- test data correctness when new datasets are introduced
-- handle a model training that is computationally intensive
+- Test data correctness when new datasets are introduced
+- Handle a model training that is computationally intensive
```

Comment:
@jorgeorpinel I understand it that it should either be:

* capitalization _and_ punctuation, or
* no capitalization and no punctuation.

Your change is a mix. Isn't it?

Reply:
It's fine to capitalize without an ending period. Optional but it's what we use so far everywhere so recommended, for consistency.